packages/explat/src/assignment.ts

```diff
@@ -18,7 +19,7 @@ export const fetchExperimentAssignment = async ( {
 		);
 	}
 
-	const params = stringify( {
+	const params = applyFilters( 'woocommerce_explat_request_args', {
```

Comment:
I think it'll be good to add some comment on this filter similar to this https://github.com/woocommerce/woocommerce-admin/blob/1946841f97b7c7aa2876362fec1651bd8a6e8215/client/homescreen/stats-overview/defaults.js#L5-L8

Reply:
That's a good idea! Updated in https://github.com/woocommerce/woocommerce-admin/pull/8231/commits/8df5fa38d49eae2dedb286796e13408c4cd1a7a3